src/Sylius/Bundle/ApiBundle/Resources/config/services/controller_locale.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
+    <services>
+        <defaults public="true" />
+
+        <service id="Sylius\Bundle\ApiBundle\Controller\Locale\DeleteLocaleAction">
+            <argument type="service" id="Sylius\Bundle\LocaleBundle\Remover\LocaleRemoverInterface" />
+
+            <tag name="controller.service_arguments" />
```

Comment:
```suggestion
```
and let's make this controller public (if we even want to leave it)

Reply:
I'll probably get rid of this controller anyway.